### PR TITLE
Authentication

### PR DESCRIPTION
**Pull request by naveen623**
I have done the 3Tasks regarding the Authenticate. Please review it.

### CODE REVIEW

File: Project1/Templates/Registration.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        Welcome to My Page
+    </title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" 
+        integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <div class="jumbotron jumbotron-fluid">
+        <div class="container">
+        <h1 class="display-4">Registration</h1>
+        </div>
+    </div>
+    <style>
```
Comment (ashishk144) [inline Project1/Templates/Registration.html]: Move CSS to stylesheets.

File: Project1/Templates/Registration.html [hunk 2]
```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>
+        Welcome to My Page
+    </title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
```
Comment (ashishk144) [inline Project1/Templates/Registration.html]: have a generic template and extend the generic template to this file.

File: Project1/Templates/User.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+            width: 20%;
+            opacity: 0.9;
+            }
+          
+       </style>
+   </head>
+   <body>
+      <h2> Succesfully Created Your Page  </h2>
+      <!-- <h3> Welcome {{f}} </h3> -->
+      <form action = "/register" >
```
Comment (ashishk144) [inline Project1/Templates/User.html]: Why is the form action /register?

File: Project1/Templates/Signin.html [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+    <body>
+
+       
+
+        <h2>Name </h2>
+        <input type="text" placeholder="Name" name = "">
+
+        <h2>Email Id</h2>
+        <input type="text" placeholder="Email Id" name = "" required>
```
Comment (ashishk144) [inline Project1/Templates/Signin.html]: Is there a requirement for this page as you've provided both login and register option in the registration page itself?

File: Project1/create.py [hunk 5]
```diff
@@ -0,0 +1,24 @@
+import os
```
Comment (ashishk144) [inline Project1/create.py:1]: Delete the file as it is unused

File: Project1/details.py [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+from flask_sqlalchemy import SQLAlchemy
+from datetime import datetime
+
+
+
+
+db = SQLAlchemy()
+class Registration(db.Model):
+    __tablename__ = "Details"
+    # Firstname = db.Column(db.String, nullable=False)
```
Comment (ashishk144) [inline Project1/details.py]: remove commented code

File: Project1/application.py [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+
+@app.route("/register",methods = ["GET","POST"])
+def register():
+    if request.method == "GET":
+        return render_template("Registration.html")
+    elif request.method == "POST":
+        name = request.form.get("fname")
+        email = request.form.get("Email")
+        pswd  = generate_password_hash(str(request.form.get("password")))
+
```
Comment (ashishk144) [inline Project1/application.py]: check if the user is already present on the db or not

File: Project1/application.py [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+        register = Registration(Email = email, Password = pswd,dt = time.ctime(time.time()))
```
Comment (ashishk144) [inline Project1/application.py]: Why is the argument dt sent when you are using datetime.now() in the constructor.

**Review by ashishk144 — COMMENTED** [review]
Why are there unrelated pages such as search page in authentication branch?

**Comment by naveen623** [comment]
I have modified as per the requirements of project 1. Please review it.

**Pull request closed without merging**